pom.xml

```diff
@@ -122,35 +122,36 @@
                 <pluginManagement>
                     <plugins>
                         <plugin>
-                            <groupId>com.spotify</groupId>
-                            <artifactId>docker-maven-plugin</artifactId>
-                            <version>${docker.plugin.version}</version>
+                            <groupId>org.codehaus.mojo</groupId>
+                            <artifactId>exec-maven-plugin</artifactId>
+                            <version>3.1.1</version>
                             <executions>
                                 <execution>
+                                    <id>docker-build</id>
                                     <phase>install</phase>
                                     <goals>
-                                        <goal>build</goal>
+                                        <goal>exec</goal>
                                     </goals>
+                                    <configuration>
+                                        <executable>${container.executable}</executable>
+                                        <workingDirectory>${docker.image.dockerfile.dir}</workingDirectory>
+                                        <arguments>
+                                            <argument>build</argument>
+                                            <argument>-f</argument>
+                                            <argument>Dockerfile</argument>
+                                            <argument>--build-arg</argument>
+                                            <argument>ARTIFACT_NAME=${project.build.finalName}</argument>
+                                            <argument>--build-arg</argument>
+                                            <argument>EXPOSED_PORT=${docker.image.exposed.port}</argument>
+                                            <argument>--build-arg</argument>
+                                            <argument>DOCKERIZE_VERSION=${docker.image.dockerize.version}</argument>
+                                            <argument>-t</argument>
+                                            <argument>${docker.image.prefix}/${project.artifactId}</argument>
+                                            <argument>${project.build.directory}</argument>
+                                        </arguments>
+                                    </configuration>
                                 </execution>
                             </executions>
-                            <configuration>
-                                <imageName>${docker.image.prefix}/${project.artifactId}</imageName>
```

Comment:
What I see is that you've replaced the use of the spotify plugin with a call to the docker CLI (or podman). 
I have the impression that you've lost some functionality, such as the ability to push the image into Docker Hub.
Am I mistaken?

Reply:
I didn’t see documentation about pushing containers to Docker Hub or other repositories. I did check for a github workflow that might do this. The spotify plugin looks retired. As the container build with this plugin requires podman or docker, those tools can easily be used for pushing. Podman compose can also be used to start the application if build with podman.

Reply:
https://github.com/spotify/docker-maven-plugin

Reply:
I don't think I wrote it in the `readme.md` but I was using this kind of command line to publish all docker images to docker hub:
```
mvn install -PbuildDocker -DpushImageTag -DdockerImageTags=2.1.2,java11
```
I know that both Spotify projects [docker-maven-plugin](https://github.com/spotify/docker-maven-plugin) and [dockerfile-maven](https://github.com/spotify/dockerfile-maven) are stoped. Maybe it's time to remove it. I could plan to use the Spring Boot buildpacks support.

Reply:
Could you please add to the readme a comment to switch from docker to podman:
```
or if you prefer podman:
./mvnw clean install -PbuildDocker -Dcontainer.executable=podman
```

Reply:
documentation has been added